datalad/plugin/tests/test_addurls.py

```diff
@@ -423,17 +423,27 @@ def test_addurls_subdataset(self, path):
         ds = Dataset(path).create(force=True)
 
         with chpwd(path):
-            ds.addurls(self.json_file, "{url}", "{subdir}//{name}")
+            for save in True, False:
+                label = "save" if save else "nosave"
+                ds.addurls(self.json_file, "{url}",
+                           "{subdir}-" + label + "//{name}",
+                           save=save)
 
-            for fname in ["foo/a", "bar/b", "foo/c"]:
-                ok_exists(fname)
+                for fname in ["foo-{}/a", "bar-{}/b", "foo-{}/c"]:
+                    ok_exists(fname.format(label))
+
+                assert_true(save ^ ds.repo.dirty)
```

Comment:
I would have also checked that commit didn't change -- may be it did save some changes ?
pretty much that `ds.repo.get_hexsha()` is the same if not save and different if save

Reply:
yep, meant to do that.  will add